site/app/models/SuperuserEmail.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace app\models;
+
+use app\libraries\Core;
+use app\models\Email;
```

Comment:
You don't need this `use`. It's implied that any class that's not given an absolute namespace is then using the namespace of this class.